client/lib/products-values/products-list.ts

```diff
@@ -108,39 +110,48 @@ export const JETPACK_PRODUCTS_LIST: Record< JetpackProductSlug, Product > = {
 		type: constants.PRODUCT_JETPACK_SCAN,
 		term: TERM_ANNUALLY,
 		bill_period: PLAN_ANNUAL_PERIOD,
-		getFeatures: (): string[] => [
-			FEATURE_SCAN_V2,
-			FEATURE_ONE_CLICK_FIX_V2,
-			FEATURE_INSTANT_EMAIL_V2,
-			FEATURE_PRIORITY_SUPPORT_V2,
-		],
+		getFeatures: (): string[] =>
+			( {
+				spp: [ FEATURE_SCAN_V2, FEATURE_ONE_CLICK_FIX_V2, FEATURE_INSTANT_EMAIL_V2 ],
+			}[ getJetpackCROActiveVersion() ] || [
```

Comment:
@mattgawarecki, calling directly `getJetpackCROActiveVersion` here is problematic, since this module is run server-side (I learnt that the hard way). Passing the variation as an argument to `getFeatures` instead is safer.

Reply:
Yes! I'm looking back at your PR #48464 as a guide on how to avoid. Thanks for pointing this out, good catch!

Reply:
I removed calls to `getJetpackCROActiveVersion()` from `products-list.ts` with my latest commit, 6ed5b9a. I was prepared to do the same in `features-list.ts`, but it seems to have been working fine all this time, so I guess it's not harmful in that file?

Reply:
Exactly. I ran `NODE_ENV=production CALYPSO_ENV=production yarn start` to make sure the server won't crash in production. It all looks good.